admin-dev/themes/new-theme/scss/components/_button.scss

```diff
@@ -1,12 +1,12 @@
 .btn {
   &.btn-action {
-    color: #576C72;
-    background-color: white;
+    color: #000;
```

Comment:
@NeOMakinG Color change is wanted  ?

Reply:
Reverted in https://github.com/PrestaShop/PrestaShop/pull/17617/commits/880b0ffdaca18f032d2cb941f37e0751b4745479